src/components/basic/DlAccordion/DlAccordion.vue

```diff
@@ -103,26 +101,48 @@ export default defineComponent({
             return this.$slots.header !== undefined
         }
     },
+    mounted() {
+        this.getContentHeight()
+    },
     methods: {
         handleClick() {
             if (this.modelValue !== null) {
                 this.$emit('update:model-value', !this.isOpen)
             }
             this.$emit(this.isOpen === true ? 'hide' : 'show')
             this.isOpen = !this.isOpen
+            this.getContentHeight()
+        },
+        getContentHeight() {
+            this.contentHeight =
+                (this.$refs['dlAccordionContent'] as HTMLDivElement)
+                    ?.offsetHeight + 'px'
+            this.$nextTick(() => {
+                this.contentHeight =
+                    (this.$refs['dlAccordionContent'] as HTMLDivElement)
+                        ?.offsetHeight + 'px'
+                setTimeout(() => {
+                    this.contentHeight = 'auto'
+                }, 300) //todo: Fix animation for the child elements
```

Comment:
please explain, what will not be supported currently?

Reply:
in nested elements
There is a slight jump in the animation starting with the first child
In my opinion, compared to the current situation, it is not so bad
I preferred not to bother with it at this point.
There are more improvements and components that I plan

I will add a video shortly

Reply:
https://github.com/dataloop-ai/components/assets/23630595/15a3c548-9b1f-4925-987c-35cd61eacc49

Reply:
@RotemDl